src/Nancy.Demo.Authentication.Forms.TestingDemo/LoginFixture.cs

```diff
@@ -17,10 +17,10 @@ public LoginFixture()
         }
 
         [Fact]
-        public void Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
+        public async Task Should_redirect_to_login_with_error_querystring_if_username_or_password_incorrect()
         {
             // Given, When
-            var response = browser.Post("/login/", (with) =>
+            var response = await browser.Post("/login/", (with) =>

```

Comment:
I'm not fond with wrapping a sync api (`.Wait` `.Result` ) around and async api at a library level. Funny things happen with deadlock and what not. See `HttpClient` 

The methods _could_ be renamed `PostAsync()` so it more of a clear breaking change?